edx_rest_api_client/client.py

```diff
@@ -12,7 +12,8 @@ def __init__(self, url, signing_key=None, username=None, full_name=None, email=N
         Instantiate a new client.
 
         Raises:
-            ValueError, if either the URL or necessary authentication values are not provided.
+            ValueError, if either the URL is not provided.
```

Comment:
Grammar. Also, `ValueError:`. The previous iteration was incorrect. https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments.